### PR TITLE
CMFPLONE#2179: images overlapping with portlets ....

### PR DESCRIPTION
**Pull request by fgrcon**
see https://github.com/plone/Products.CMFPlone/issues/2179

### CODE REVIEW

File: plonetheme/barceloneta/theme/less/image.plone.less [hunk 1]
```diff
@@ -19,4 +19,10 @@
...
 	border-radius: 50px;
-}
\ No newline at end of file
+}
+
+// resize image in image.pt when portlets exist ....
+.image-product img {
+    width: 100%;
+    height: 100%;
+}
```
Comment (agitator) [inline plonetheme/barceloneta/theme/less/image.plone.less]: shouldn't be 
```
max-width: 100%;
height: auto;
``` 
enough?

File: plonetheme/barceloneta/theme/less/main.plone.less [hunk 2]
```diff
@@ -19,7 +19,9 @@
 
 //*// Adjustments
 #content {margin-bottom: @plone-padding-base-horizontal * 4;}
-
+#content-core {
+  overflow: hidden;
+}
```
Comment (agitator) [inline plonetheme/barceloneta/theme/less/main.plone.less]: why this?

**Comment by jensens** [comment]
@fgrcon Any update on this? (see questions above)

**Comment by fgrcon** [comment]
@agitator, @jensens  :  Thx - should be fixed now ....

**Pull request merged**